osm2gtfs/tests/accra_unittests.py

```diff
@@ -100,8 +100,8 @@ def test_refresh_routes_cache(self):
         self.assertEqual(len(routes), 277, 'Wrong count of routes in the cache file')
```

Comment:
you need to change the name of the cache file here too (`routes-accra` to `accra-routes`)